projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.ts

```diff
@@ -181,11 +181,17 @@ export class QuotesComponent implements OnInit, OnDestroy {
   protected setCategoryDropdown() {
     this.options = [];
     this.subscription.add(
-      this.translation
-        .translate('quote.allQuotes')
+      combineLatest([
+        this.translation.translate('quote.allQuotes'),
+        this.translation.translate('quote.renewalQuotes'),
+      ])
         .pipe(
-          map(firstOption => {
+          tap(([firstOption, renewalOption]) => {
             this.options.push({ name: firstOption, code: '' });
+            const renewalQuote = this.quotes.find(quote => quote.renewal);
+            if (renewalQuote) {
+              this.options.push({ name: renewalOption, code: true });
```

Comment:
Here I extend Category dropdown with the Renewal Quotes label to allow filtering by that criteria.